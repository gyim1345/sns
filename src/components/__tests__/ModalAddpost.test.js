import React from 'react';
import { mount } from 'enzyme';
import ModalAddpost from '../ModalAddpost';
import { currentUser } from './testjs';
import { MemoryRouter } from 'react-router-dom';

describe('<ModalAddpost />', () => {
  it('스냅샷 비교', () => {
    const wrapper = mount(
      <MemoryRouter
        initialEntries={[{ pathname: '/gibong@gmail.com', key: 'testKey' }]}
      >
        <ModalAddpost currentUser={currentUser} />
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
