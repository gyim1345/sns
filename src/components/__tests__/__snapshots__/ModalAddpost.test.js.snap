// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalAddpost /> 스냅샷 비교 1`] = `
<MemoryRouter
  initialEntries={
    Array [
      Object {
        "key": "testKey",
        "pathname": "/gibong@gmail.com",
      },
    ]
  }
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "key": "testKey",
            "pathname": "/gibong@gmail.com",
            "search": "",
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "key": "testKey",
          "pathname": "/gibong@gmail.com",
          "search": "",
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <ModalAddPost
      currentUser="gibong@gmail.com"
    />
  </Router>
</MemoryRouter>
`;
